README.md

```diff
@@ -46,20 +46,41 @@ Now you can start by running a baseline either by looking into [the last results
 
 ## Evaluation using the COCO API
 
+
+### DISCLAIMER: New Dataset! 
+
+To account for the new 1 Megapixel Dataset described in our recently accepted NeurIPS submission: "Learning to detect 1 Megapixel Event Camera", the format has slightly changed. 
```

Comment:
paper title is "Learning to Detect Objects with a 1 Megapixel Event Camera"

Reply:
ah thanks yes will change that